Dockerfile.build.alpine

```diff
@@ -1,9 +1,13 @@
-FROM gliderlabs/alpine:3.2
-MAINTAINER Jussi Nummelin <jussi.nummelin@gmail.com>
-RUN apk-install bash go bzr git mercurial subversion openssh-client ca-certificates
+FROM alpine:3.3
+MAINTAINER Rene Kaufmann <kaufmann.r@gmail.com>
```

Comment:
We should just remove this MAINTAINER tag or everyone's just going to be updating it every time a patch is made.